packages/flex-plugin-e2e-tests/src/tests/step009.ts

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable */
+import { TestSuite, TestParams } from '..';
+import { api, Browser, ConsoleAPI, pluginHelper } from '../utils';
+
+const PLUGIN_RELEASED_TIMEOUT = 30000;
+const PLUGIN_RELEASED_POLL_INTERVAL = 5000;
+
+// Plugin visible on the Hosted Flex
+const testSuite: TestSuite = async (params: TestParams): Promise<void> => {  
+  // Fetch and find latest released plugin  
+  const release = await api.getActiveRelease();
+  const plugins = await api.getActivePlugins(release!.configuration_sid);
+
+  const releasedPlugin = plugins.plugins.find(plugin => plugin.unique_name == params.plugin.name);
+
+  if(!releasedPlugin) {
+    throw new Error(`Did not find plugin with name: ${params.plugin.name} in released plugins`);
+  }   
+
+  const consoleApi = new ConsoleAPI(params.consoleBaseUrl);
+  const cookies = await consoleApi.getCookies();
+  
+  await Browser.create();
+
+  try {
+    // Log into Flex
+    await Browser.loginViaConsole(cookies, params.consoleBaseUrl, params.hostedFlexBaseUrl);
+    await Browser.userIsOnView('Admin Dashboard');
+
+    // Make sure that /plugins contain the plugin
+    await pluginHelper.waitForPluginToRelease(params.hostedFlexBaseUrl, releasedPlugin, PLUGIN_RELEASED_TIMEOUT, PLUGIN_RELEASED_POLL_INTERVAL);
+
+    await Browser.navigate(params.hostedFlexBaseUrl, 'agent-desktop');
+
+    if(!params.plugin.newlineValue) {
+        throw new Error(`params.plugin.newlineValue does not have a valid value`);
+    }
+
+    await Browser.pluginIsVisible(params.plugin.newlineValue);
+
+  } catch(e) {
+    await Browser.kill();
```

Comment:
Use `finally`- I _think_ it is supported now:

```js
try {
  ...
} finally {
  await Browser.kill();
}
```

Note: No `catch` block because we want to throw the exception. And `finally` will be called even if exception is thrown.